binding.gyp

```diff
@@ -13,53 +12,76 @@
       ],
       'include_dirs': [
         'blst/bindings',
-        "<!@(node -p \"require('node-addon-api').include_dir\")",
+        '<!@(node -p \"require("node-addon-api").include\")',
+      ],
+      'dependencies': [
+        '<!(node -p \"require("node-addon-api").gyp\")'
       ],
-      'dependencies': [ "<!(node -p \"require('node-addon-api').gyp\")" ],
       'defines': [ 
         'NAPI_CPP_EXCEPTIONS'
       ],
       'cflags': [
-          '-fexceptions',
-          '-Werror',
-          '-Wall',
-          '-Wextra',
-          '-Wpedantic',
+        '-fno-builtin-memcpy',
+        '-fexceptions',
+        '-Werror',
+        '-Wall',
+        '-Wextra',
+        '-Wpedantic',
       ],
       'cflags_cc': [
-          '-std=c++17',
```

Comment:
Does removing this flag means it uses any available C++ version?

Reply:
Yes and we need a minimum of C++17 for some of the features we are using.  Newer compilers are always backwards compatible but older ones throw hard errors for newer versions.  I had a pretty long discussion about this with Hubert and 17 is in ubuntu 18 so realistically we are supporting enough "ancient" architectures to cover 99.9999% of potential consumers.